products/llbuildSwift/CoreBindings.swift

```diff
@@ -374,7 +420,7 @@ public class BuildEngine {
     /// The number of rules which have been defined.
     public var numRules: Int = 0
 
-    public init(delegate: BuildEngineDelegate) {
+    public init(delegate: BuildEngineDelegate, serial: Bool = false, schedulerAlgorithm: SchedulerAlgorithm = .commandNamePriority, schedulerLanes: UInt32 = 0) {
```

Comment:
This is a change in functionlity, since before Core was getting the serial scheduler. At the same time, spawn APIs were not available before, so it won't change how current users process their rules.